<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtStochasticityInspectorHlrtWidget</class>
 <widget class="QWidget" name="QtStochasticityInspectorHlrtWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_dof">
     <property name="text">
      <string>Degrees of freedom</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_delta">
     <property name="text">
      <string>delta</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="box_dof">
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLineEdit" name="edit_l0">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="edit_ll1">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_alpha">
     <property name="text">
      <string>Alpha</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLineEdit" name="edit_l1">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_l0_2">
     <property name="text">
      <string>Likelihood BM</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="label_conclusion">
     <property name="text">
      <string>Conclusion: [here]</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="edit_ll0">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_l0">
     <property name="text">
      <string>Log-likelihood BM</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLineEdit" name="edit_delta">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_l1_2">
     <property name="text">
      <string>Likelihood OU</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_l1">
     <property name="text">
      <string>Log-likelihood OU</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="box_alpha">
     <property name="maximum">
      <double>0.100000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_critical_value">
     <property name="text">
      <string>Critical value</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="edit_critical_value">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>box_alpha</tabstop>
  <tabstop>box_dof</tabstop>
  <tabstop>edit_critical_value</tabstop>
  <tabstop>edit_ll0</tabstop>
  <tabstop>edit_ll1</tabstop>
  <tabstop>edit_l0</tabstop>
  <tabstop>edit_l1</tabstop>
  <tabstop>edit_delta</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
